addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -927,7 +940,21 @@ options.registry.OptionsTab = options.Class.extend({
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
-
+    /**
+     * @private
+     * @param {String} id
+     * @param {Object} settings: gray settings that will be adjusted to
+     * @returns {String} the new color of gray
+     */
+    _buildGray(id, settings) {
+        this['gray-hue'] = settings['gray-hue'] || this['gray-hue'];
+        this['gray-extra-saturation'] = settings['gray-extra-saturation'] || this['gray-extra-saturation'];
```

Comment:
Strange to have to do that for each gray when you update, I think this could be refactored a bit to be prettier.